crossbuilder

```diff
@@ -662,9 +661,7 @@ check_for_container_network() {
     NETWORK_UP=0
     for i in `seq 1 10`
     do
-        if lxc query "/1.0/containers/${LXD_CONTAINER}/state" | \
-                jq -e '.network.eth0.addresses | any( .family == "inet" )' \
-                > /dev/null 2>&1 ; then
+        if lxc query "/1.0/containers/${LXD_CONTAINER}/state" | jq -e '.network.eth0.addresses | any( .family == "inet" )' > /dev/null 2>&1 ; then
```

Comment:
also this one, please let it be in multiple lines :-)

Reply:
I agree, this one is longer, but still not the longest. There are 6 lines longer than this and not splited. So why this and not the other? Also the multi-line version is inconsistently indented too, and there is output redirection alone on the third line, which can mislead for which command it is for.

Reply:
I'm not very consistent on this, admittedly; but if we are making cosmetic style changes, let's go towards shorter lines please. :-)

Extra points if you propose splitting the other long lines too :-)

Reply:
If I may ask again? How do you differentiate between long and short lines, where is the line, where the short line becomes to long?

For me speaking, no line is too long. I use editor, which wraps lines. When I split, it is to improve readability of complicated 'if' statements. Also more lines will be shorter if the indentation is not 4 spaces (on this my opinion is to use 'tab' and anyone can set editor to his liking on how much spaces will be used for tabs).

Reply:
A line width of 79 chars max is the most safe, and people can rely on that to keep multiple editor windows opened side by side without having automatic line wrap. Yes, most editors have automatic line wrapping, but it's still easier to read code where lines are split manually.

Reply:
So you want from me to keep the lines most 79 characters long and you introduced more longer lines few commits ago (eg. [this one](https://github.com/ubports/crossbuilder/commit/cd781279ba9b56d9a6e4ffe4f9737ab66a434c46#diff-4aa8b5f5a0460602bde364cf47e137d5692e5d61b00484d3667cc3d881fc8dfcR564)? This lacks consistency from your side.